micro-rdk/src/common/webrtc/candidates.rs

```diff
@@ -6,6 +6,19 @@ use std::{
 };
 
 use stun_codec::TransactionId;
+use thiserror::Error;
+
+#[derive(Error, Debug, PartialEq, Eq)]
+pub enum CandidateError {
+    #[error("cannot parse candidate")]
```

Comment:
can we keep the printing of the candidate string in the error?

Reply:
I rather not allocate a String for that it happens somewhat often and is not a critical failure